src/autogluon/bench/frameworks/tabular/tabular_benchmark.py

```diff
@@ -11,6 +11,25 @@
 
 
 class TabularBenchmark(Benchmark):
+    """A benchmark class for AutoGluon Tabular.
+
+    Args:
+        benchmark_name (str): The name of the benchmark.
+        root_dir (str): The root directory for storing benchmark results (default: "./benchmark_runs/tabular/").
+
+    Attributes:
+        benchmark_name (str): The name of the benchmark.
+        root_dir (str): The root directory for storing benchmark results.
+        module (str): The name of the benchmark module (in this case, "tabular").
+
+    Methods:
+        setup(): Sets up the benchmark environment.
+        run(): Runs the benchmark.
+
+    Raises:
+        subprocess.CalledProcessError: If the setup or execution scripts encounter an error.
```

Comment:
Similarly, I'm not sure it makes sense to say a class "Raises". If we want to document this, should it be with the individual functions that might raise the error?